test/org/opensolaris/opengrok/web/StatisticsTest.java

```diff
@@ -94,28 +94,6 @@ public void testMaybeRefresh() {
         }
     }
 
-    /*
-    @Test
-    public void testAddRequest_HttpServletRequest() {
-        System.out.println("addRequest");
-        HttpServletRequest req = null;
-        Statistics instance = new Statistics();
-        instance.addRequest(req);
-        // TODO review the generated test code and remove the default call to fail.
-        Assert.fail("The test case is a prototype.");
-    }
-
-    @Test
-    public void testAddRequest_HttpServletRequest_String() {
-        System.out.println("addRequest");
-        HttpServletRequest req = null;
-        String category = "";
-        Statistics instance = new Statistics();
-        instance.addRequest(req, category);
-        // TODO review the generated test code and remove the default call to fail.
-        fail("The test case is a prototype.");
-    }
-     */
```

Comment:
There was this auto generated test section from NB which I think I forgot to remove last time.